source/includes/na/take_payments/_payment_3d_secure.md

```diff
@@ -5,20 +5,37 @@ Definition
 POST /v1/payments HTTP/1.1
 
 Request
-curl https://api.na.bambora.com/v1/payments
--H "Authorization: Passcode MzAwMjAwNTc4OjRCYU..."
--H "Content-Type: application/json"
+curl https://api.na.bambora.com/v1/payments \
+-H "Authorization: Passcode MzAwMjAwNTc4OjRCYUQ4MkQ5MTk3YjRjYzRiNzBhMjIxOTExZUU5Zjcw" \
```

Comment:
Is there a reason we've expanded this to a full passcode as opposed to one suffixed with the '...' as before? Probably doesn't matter but might be better to keep it consistent with what we had before, unless it was changed for a specific reason

Reply:
No specific reason, I'll add that to the list of non-urgent fixes :)